contrib/tools/workflowcheck/workflow/checker.go

```diff
@@ -152,10 +152,21 @@ func (c *Checker) Run(pass *analysis.Pass) error {
 	lookupCache := determinism.NewPackageLookupCache(pass)
 	// Check every register workflow invocation
 	for _, file := range pass.Files {
+		// Get ignore map for this file
+		ignoreMap := map[ast.Node]struct{}{}
+		determinism.UpdateIgnoreMap(pass.Fset, file, ignoreMap)
+
 		ast.Inspect(file, func(n ast.Node) bool {
-			// Only handle calls
+			// Only handle calls with followable function pointers
+			_, isIgnored := ignoreMap[n]
+			for k := range ignoreMap {
+				asExprStmt, _ := k.(*ast.ExprStmt)
+				if asExprStmt != nil && asExprStmt.X == n {
+					isIgnored = true
+				}
+			}
```

Comment:
In retrospect, you could have just:

```suggestion
			if _, isIgnored := ignoreMap[n]; isIgnored {
				return false
			}
```

Reply:
That didn't seem to work when I was trying it - I needed to get this inner node for things to work